nexus/sites.py

```diff
@@ -7,12 +7,12 @@
 from django.template import RequestContext
 from django.template.loader import render_to_string
 from django.utils.datastructures import SortedDict
-from django.utils.functional import update_wrapper
```

Comment:
As of Django 1.5, `django.utils.functional.update_wrapper` [is just a reexport from functools](https://github.com/django/django/blob/1.5.12/django/utils/functional.py#L3). Since 1.6, `update_wrapper` is not longer included in `django.utils.functional`